pkg/dotc1z/manager/local/local.go

```diff
@@ -84,7 +84,7 @@ func (l *localManager) LoadC1Z(ctx context.Context) (*dotc1z.C1File, error) {
 		zap.String("temp_path", l.tmpPath),
 	)
 
-	return dotc1z.NewC1ZFile(ctx, l.tmpPath, dotc1z.WithTmpDir(l.tmpDir))
+	return dotc1z.NewC1ZFile(ctx, l.tmpPath, dotc1z.WithTmpDir(l.tmpDir), dotc1z.WithPragma("journal_mode", "WAL"))
```

Comment:
hmm, are there any side effects with this?  When we re-compress the sqlite data file, wouldn't we also need the WAL file(s)?

Reply:
The WAL file is deleted when the last DB connection is closed. We always close the DB before calling saveC1z(), so the WAL file doesn't exist when we compress. If I comment out the calls to os.Remove() and os.RemoveAll(), the only file in the temp dir is the DB.

```
ggreer@gmac:/var/folders/by/00zvpw316q50n2r1jlwqkqgr0000gn/T/c1z2968114110% ls -la
total 18880
drwx------    3 ggreer  staff       96 Nov 22 11:16 ./
drwx------@ 570 ggreer  staff    18240 Nov 22 11:16 ../
-rw-r--r--    1 ggreer  staff  9666560 Nov 22 11:16 db
```